package.json

```diff
@@ -10,6 +10,7 @@
   "license": "ISC",
   "dependencies": {
     "express": "^4.16.4",
+    "mongo-client": "^0.2.1",
```

Comment:
Remove mongo-client from package.json, the project is using mongodb already. This is not being used at all.